src/System Application/App/Azure Blob Services API/src/ABSBlobClient.Codeunit.al

```diff
@@ -1114,4 +1114,43 @@ codeunit 9053 "ABS Blob Client"
 
     var
         ABSClientImpl: Codeunit "ABS Client Impl.";
+
+    /// <summary>
+    /// The Get Blob MetaData operation gets user-defined MetaDatas for the specified blob as one or more key-value pairs.
+    /// see: https://learn.microsoft.com/en-us/rest/api/storageservices/get-blob-metadata?tabs=microsoft-entra-id
+    /// </summary>
+    /// <param name="BlobName">The name of the blob.</param>
+    /// <param name="MetaData">The result Dictionary of [Text, Text] with blob metadatas.</param>
+    /// <returns>An operation response object</returns>
+    procedure GetBlobMetaData(BlobName: Text; var MetaData: Dictionary of [Text, Text]): Codeunit "ABS Operation Response"
+    var
+        OptionalParameters: Codeunit "ABS Optional Parameters";
+    begin
+        exit(ABSClientImpl.GetBlobMetaData(BlobName, MetaData, OptionalParameters))
+    end;
+
+    /// <summary>
+    /// The Get Blob MetaData operation gets user-defined MetaDatas for the specified blob as one or more key-value pairs.
```

Comment:
please apply the comment suggestions from above to all objects. 
Metadata -> 1 word, no plurals
procedure names should be used as such -> The GetBlobMetadata operation etc...

Reply:
Updated , Modified the Procedure name and other comments